source/Application-Starter/ApplicationStarterCommandLineHandler.class.st

```diff
@@ -139,94 +150,22 @@ ApplicationStarterCommandLineHandler >> isDebugModeEnabled [
 	^ self hasOption: 'debug-mode'
 ]
 
-{ #category : #'private - logging' }
-ApplicationStarterCommandLineHandler >> log: aMessage to: anOutputStream withLevel: aLogLevel [
-
-	anOutputStream
-		nextPutAll: ( '[<1p>] [<2s>] <3s><n>' expandMacrosWith: DateAndTime current with: aLogLevel with: aMessage );
-		flush
-]
-
-{ #category : #'private - logging' }
-ApplicationStarterCommandLineHandler >> log: aMessage to: anOutputStream withLevel: aLogLevel during: aBlock [
-
-	self log: aMessage to: anOutputStream withLevel: aLogLevel.
-	[ aBlock value.
-	self log: aMessage , '... [OK]' to: anOutputStream withLevel: aLogLevel
-	]
-		ifCurtailed: [ self logError: aMessage , '... [FAILED]' ]
-]
-
-{ #category : #'private - logging' }
+{ #category : #'private - activation' }
 ApplicationStarterCommandLineHandler >> logConfiguration [
 
 	| configuration |
 
 	configuration := self configuration.
 	configuration keys sorted
 		do: [ :configurationKey | 
-			self
-				logInfo:
+			CurrentLogger value
+				info:
 					( '<1s>: <2p>'
 						expandMacrosWith: configurationKey asString
 						with: ( configuration at: configurationKey ) )
 			]
 ]
 
-{ #category : #'private - logging' }
-ApplicationStarterCommandLineHandler >> logError: anErrorMessage [
-
-	self log: anErrorMessage to: self stderr withLevel: 'ERROR'
-]
-
-{ #category : #'private - logging' }
-ApplicationStarterCommandLineHandler >> logError: anErrorMessage during: aBlock [
-
-	self
-		log: anErrorMessage
-		to: self stderr
-		withLevel: 'ERROR'
-		during: aBlock
-]
-
-{ #category : #'private - logging' }
-ApplicationStarterCommandLineHandler >> logInfo: aMessage [
-
-	self log: aMessage to: self stdout withLevel: 'INFO'
-]
-
-{ #category : #'private - logging' }
-ApplicationStarterCommandLineHandler >> logInfo: anErrorMessage during: aBlock [
```

Comment:
Erasing this messages breaks compatibility and every application starter using it